contracts/interfaces/IMechMarketplace.sol

```diff
@@ -1,24 +1,24 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.28;
 
+import {DeliverWithSignature} from "../OlasMech.sol";
+
 /// @dev Mech Marketplace interface
 interface IMechMarketplace {
     /// @dev Delivers requests.
     /// @notice This function can only be called by the mech delivering the request.
     /// @param requestIds Set of request ids.
     /// @param mechDeliveryRates Corresponding set of actual charged delivery rates for each request.
-    /// @param deliveryDatas Set of corresponding self-descriptive opaque delivery data-blobs.
-    function deliverMarketplace(bytes32[] memory requestIds, uint256[] memory mechDeliveryRates,
-        bytes[] memory deliveryDatas) external returns (bool[] memory deliveredRequests);
+    /// @return deliveredRequests Corresponding set of successful / failed deliveries.
+    function deliverMarketplace(bytes32[] calldata requestIds, uint256[] memory mechDeliveryRates)
+        external returns (bool[] calldata deliveredRequests);
```

Comment:
the return paramter should be:

```solidity
bool[] memory deliveredRequests
```

Reply:
Thanks, sorry for the confusion, but it got fixed later, here it was just a batch replace in files